pkg/operator/operator.go

```diff
@@ -475,6 +472,5 @@ func buildPolicyLoader(tc trivyoperator.ConfigData) (policy.Loader, error) {
 	}
 	ro.Insecure = tc.PolicyBundleInsecure()
 	artifact.RegistryOptions = ro
-	policyLoader := policy.NewPolicyLoader(tc.PolicyBundleOciRef(), gcache.New(1).LRU().Build(), ro, mp.WithOCIArtifact(artifact))
-	return policyLoader, nil
+	return policy.NewPolicyLoader(tc.PolicyBundleOciRef(), gcache.New(2).LRU().Build(), ro, mp.WithOCIArtifact(artifact))
```

Comment:
I'm curious why is the size 2 now? What does the size depend upon?

Reply:
it was interesting to me too.
this size is max count of items in cache - https://github.com/bluele/gcache/blob/master/lru.go#L46
but we try to keep 2 items there: `bundlePolicies` and `bundlePath`:
https://github.com/aquasecurity/trivy-operator/blob/5d60ec9e3205b951fd5e9b6f75f7fcd9d2c2d642/pkg/policy/loader.go#L105-L106

`bundlePath` is the second, so `bundlePolicies` never saves now